app/frontend/src/common/authenticate.js

```diff
@@ -45,31 +46,40 @@ export default {
         resolve() // We've already authenticated, have a header, and we've not expired.
       } else {
         // Attempt to retrieve a stored token, this may avoid us having to refresh the page.
-        const token = localStorage.getItem('token')
-        const refreshToken = localStorage.getItem('refreshToken')
-        const idToken = localStorage.getItem('idToken')
+
+        store.commit('SET_KEYCLOAK_READY', false)
```

Comment:
You may not need the SET_KEYCLOAK_READY state anymore (see comments below re. authentication now resolving promise correctly).